subsys/bluetooth/services/Kconfig

```diff
@@ -0,0 +1,17 @@
+# Kconfig - GATT Services
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BT_CONN
+
+menu "GATT Services"
+
+source "subsys/bluetooth/services/Kconfig.bas"
+
+endmenu
+
+endif #BT_PERIPHERAL
```

Comment:
This doesn't match the actual contition, should be `BT_CONN` (I know this is because it used to be different in an earlier revision)

Reply:
Fixed